app/src/main/res/values/strings.xml

```diff
@@ -89,4 +89,9 @@
     <string name="no_topic_in_message">(no topic)</string>
     <string name="message_copied">Message Copied</string>
     <string name="login_failed">Unable to login</string>
+    <string name="new_message_mul_sender" formatted="false">%d New Message%s</string>
+    <string name="new_message" formatted="false">%d New Message%s %s</string>
```

Comment:
It's best practice to number the strings that are being replaced, eg `<string name="welcome_messages">Hello, %1$s! You have %2$d new messages.</string>`

See Formatting Strings in https://developer.android.com/guide/topics/resources/string-resource.html